api/src/main/java/org/pmiops/workbench/db/dao/UserDao.java

```diff
@@ -22,4 +24,8 @@
    */
   @Query("SELECT user FROM User user LEFT JOIN FETCH user.authorities WHERE user.userId = :id")
   User findUserWithAuthorities(@Param("id") long id);
+
+  @Query("SELECT DISTINCT user.freeTierBillingProjectName FROM User user\n" +
+      "WHERE user.freeTierBillingProjectName IS NOT NULL")
+  Set<String> getAllUserProjects();
```

Comment:
This eventually could return a lot of data. I'm not sure whether it will cause problems. It won't happen anytime soon, so probably nothing to worry about for now... just noting.